src/include/type/expression/comparison_expression.h

```diff
@@ -0,0 +1,33 @@
+#pragma once
+
+#include <vector>
+#include "type/expression/abstract_expression.h"
+#include "type/expression/expression_defs.h"
+#include "type/type_id.h"
+
+namespace terrier {
+namespace type {
+namespace expression {
```

Comment:
@tli2 where did we land on namespace depth? Are we making a global change to allow more, or just an exception for the execution folder?